src/curated-corpus/pages/ProspectingPage/ProspectingPage.tsx

```diff
@@ -595,6 +600,14 @@ export const ProspectingPage: React.FC = (): JSX.Element => {
         />
       )}
 
+      {approvedItem && (
+        <DuplicateProspectModal
+          approvedItem={approvedItem}
+          isOpen={duplicateProspectModalOpen}
+          toggleModal={toggleDuplicateProspectModal}
+        />
+      )}
+
```

Comment:
This is the bug fix.  Previously I had:

```
 {approvedItem && (
        <ScheduleItemModal ...  ) && ( <DuplicateProspectModal ... )
```

Which was only rendering the `<DuplicateProspectModal>`. I broke it down into two separate conditionals